src/views/AnnualReport.vue

```diff
@@ -208,15 +208,15 @@
       <div class="buttons-left">
         <v-btn id="ar-save-btn" large
           v-if="isAnnualReportEditable"
-          :disabled="!isSaveButtonEnabled || busySaving"
```

Comment:
Just curious how this ties in here?

Reply:
I had to think about this for Correction filing so I asked Scott what the requirement is and whether it applies to all filings. He said you should always be allows to save a draft, for any filing. So I updated it.

Reply:
I will be sure to add test notes to the ticket so that QA knows to look at it.